### PR TITLE
feat: add faqs page

### PR DESCRIPTION
**Pull request by danielsp45**

### CODE REVIEW

File: src/components/intro.astro [hunk 1]
```diff
@@ -33,11 +33,17 @@ import ContentSection from "~/components/content-section.astro";
...
-  <div class="w-full max-w-xl lg:max-w-6xl text-left text-xl tracking-wide uppercase">
+  <div
+    class="w-full max-w-xl lg:max-w-6xl text-left text-xl tracking-wide uppercase"
+  >
     <p>The second edition will be happening from</p>
-    <p class="text-4xl text-primary">5<span class="text-2xl">TH</span> TO 7<span class="text-2xl">TH</span> APRIL 2024</p>
+    <p class="text-4xl text-primary">
+      5<span class="text-2xl">TH</span> TO 7<span class="text-2xl">TH</span> APRIL
+      2024
+    </p>
```
Comment (MarioRodrigues10) [inline src/components/intro.astro:43]: Use uppercase

File: src/components/faqs.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+              <Disclosure as="div" key={faq.question} className="pt-6">
+                {({ open }) => (
+                  <>
+                    <dt>
+                      <Disclosure.Button className="flex w-full items-start justify-start text-left text-white">
+                        <div>
+                          {open ? (
+                            <svg fill="#66B22E" className="mt-2 mr-3 w-4 h-4 rotate-90" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                              <path d="M4.5 5.653c0-1.427 1.529-2.33 2.779-1.643l11.54 6.347c1.295.712 1.295 2.573 0 3.286L7.28 19.99c-1.25.687-2.779-.217-2.779-1.643V5.653Z" clipRule="evenodd" />
+                            </svg>
```
Comment (MarioRodrigues10) [inline src/components/faqs.jsx:76]: Use an HeroIcon here instead or remove the library.

File: src/components/faqs.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                          ) : (
+                            <svg fill="#66B22E" className="mt-2 mr-3 w-4 h-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                              <path d="M4.5 5.653c0-1.427 1.529-2.33 2.779-1.643l11.54 6.347c1.295.712 1.295 2.573 0 3.286L7.28 19.99c-1.25.687-2.779-.217-2.779-1.643V5.653Z" clipRule="evenodd" />
+                            </svg>
```
Comment (MarioRodrigues10) [inline src/components/faqs.jsx:80]: Use an HeroIcon here instead or remove the library.

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by joaodiaslobo — CHANGES_REQUESTED** [review]
In order to maintain coherent code standards on all of our repos, it would be better if the FAQ questions were placed on a JSON file instead of being hardcoded.
Something like what is done on [seium.org](https://github.com/cesium/seium.org/blob/main/data/faqs.json)

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
It would be nice to have a smooth animation when opening a faq question.
You can follow [this](https://github.com/cesium/seium.org/blob/main/layout/FAQs/components/Question/index.tsx) example

**Review by joaodiaslobo — APPROVED** [review]
🪲💚

**Pull request merged**